src/utils/command.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Helper function to get the command arguments
+ *
+ * @param {string} cmd - Command name
+ * @param {string} message - The original message
```

Comment:
you specified this as a `string`, yet you supplied `context` on other files. if this is context, it should have `telegraf.Context` type.

Reply:
WKWKW lupa belom ganti dokumentasinya